packages/core/src/makeGetEndowmentsForConfig.js

```diff
@@ -171,7 +171,13 @@ function makeGetEndowmentsForConfig ({ createFunctionWrapper }) {
         const receiver = this
         // replace the "receiver" value if it points to fake parent
         const receiverRef = receiver === sourceGlobal ? targetGlobal : receiver
-        const result = Reflect.apply(sourcePropDesc.get, receiverRef, [])
+        const desc = Reflect.getOwnPropertyDescriptor(targetGlobal, 'chrome')
+        let result
+        if ('get' in desc) {
+          result = Reflect.apply(sourcePropDesc.get, receiverRef, [])
+        } else {
+          result = Reflect.getOwnPropertyDescriptor(targetGlobal, 'chrome').value
+        }
```

Comment:
`targetGlobal` should prolly be `sourceGlobal`.
should not be a hard-coded `'chrome'`
if we're getting the property descriptor again, we should use that new property descriptor

I suppose we could actually just do 
```js
Reflect.get(sourceGlobal, key, receiverRef)
```
and skip doing the prop desc lookup + if statement. one liner

Reply:
should add a comment as to why we're doing this lookup again
"sometimes getters replace themselves with static properties"